### PR TITLE
[9.0] extract analytic_department from OCA/department

### PR DESCRIPTION
**Pull request by yvaucher**
Move https://github.com/OCA/department/pull/54 here

Depends on
- [x] https://github.com/OCA/account-analytic/pull/57

### CODE REVIEW

File: analytic_department/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2011 Joël Grand-Guillaume (Camptocamp)
+# © 2013 Daniel Reis (Camptocamp)
```
Comment (dreispt) [inline analytic_department/__openerp__.py]: Not from Camptocamp

File: analytic_department/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Analytic Department Categorization",
+    "version": "8.0.1.0.0",
```
Comment (dreispt) [inline analytic_department/__openerp__.py]: 9.0.1.0.0

File: analytic_department/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    "author": "Camptocamp, Daniel Reis,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Generic Modules/Projects & Services",
+    "description": """\
```
Comment (dreispt) [inline analytic_department/__openerp__.py]: Move to readme

**Comment by yvaucher** [comment]
@dreispt Fixed, sorry for the bad copy paste of the copyright. I was not trying to assimilate you. :wink:

**Comment by dreispt** [comment]
LGTM :+1:

**Comment by moylop260** [comment]
👍

**Comment by dreispt** [comment]
Travis is green, merging.

**Pull request merged**